src/App.tsx

```diff
@@ -28,6 +28,8 @@ import IconButton from "./components/IconButton";
 import SpinBox from "./components/SpinBox";
 import HoverTooltip from "./components/HoverTooltip";
 import Export from "./components/Export";
+import DrawModeDropdown from "./components/DrawModeDropdown";
+import { Color } from "three";
```

Comment:
World's smallest nit: move this external import up to live with the react/antd imports